hbase-balancer/src/main/java/org/apache/hadoop/hbase/master/balancer/CostFunction.java

```diff
@@ -83,6 +83,10 @@ protected void regionMoved(int region, int oldServer, int newServer) {
 
   protected abstract double cost();
 
+  protected void updateWeight(double[] weights) {
```

Comment:
Where do we call this method?

Reply:
added the missing commits lost in rebase.